GothicVRProject/GothicVR
# Async loading of world

# To test
1. Start game
2. Look how performance (frames) are when loading screen is there
3. Check performance once loading is done and world is availably
4. Move to another world (e.g. oldmine)
5. Move back to main world
6. Does everything work? Performance is fine? And you can still walk and grab things?


# Checklist
(Please judge for yourself which entry is valuable for your PR)

## Scene

# Testing
